.drone.yml

```diff
@@ -0,0 +1,11 @@
+image: node0.10
```

Comment:
node 5.x is a [report-card check](https://github.com/Clever/report-card/pull/17) .. i'd be happy to enable for this repo!